src/main/resources/static/styles.css

```diff
@@ -1,3 +1,161 @@
-h1 {
-    color: blue
-}
\ No newline at end of file
+/* Stile generale */
+body {
+    font-family: Montserrat, sans-serif;
+    color: #333;
+    background-color: #f4f4f9; /* Colore di sfondo originale */
```

Comment:
i comenti per favore in inglese. Cerchiamo di tenere tutto il source code solamente in inglese.